### PR TITLE
consolidate global settings and make neccessary edits in dependent files

### PR DESCRIPTION
**Pull request by deadlight**
Done:
updated global settings, made some of the variables more semantic, removed silly ones

QA:
gulp build

### CODE REVIEW

File: scss/_global-settings.scss [hunk 1]
```diff
@@ -8,20 +8,14 @@ $asset-path: '//assets.ubuntu.com/sites/ubuntu/latest/u/';
...
-/// dark aubergine (enterprise)
-$dark-aubergine:        #2c001e !default;
-/// warm grey
-$warm-grey:             #888 !default;
+/// light brand colour
+$brand-color-light:     lighten($brand-color, 48%); ///TODO make this a lighter brand color
 /// cool grey
 $cool-grey:             #333 !default;
+/// text colour
+$text-color:			$cool-grey !default;
```
Comment (anthonydillon) [inline scss/_global-settings.scss]: Looks like a tabbing issue here.

File: scss/_global-settings.scss [hunk 2]
```diff
@@ -33,25 +27,15 @@ $black:                 #000 !default;
...
+/// error notifications
 $error:                 #df382c !default;
-/// yellow notifications
+/// warning notifications
 $warning:               #eca918 !default;
-/// green notifications
+/// success notifications
 $success:               #38b44a !default;
-/// cyan notifications
+/// information
```
Comment (anthonydillon) [inline scss/_global-settings.scss]: Can this please be information notifications please.

File: scss/_global-settings.scss [hunk 3]
```diff
@@ -33,25 +27,15 @@ $black:                 #000 !default;
...
-$status-red:            #df382c !default;
-/// grey, for disabled status or ones that don’t require attention
-$status-grey:           #888 !default;
-/// yellow, for status that require attention
-$status-yellow:         #efb73e !default;
-/// blue, for status that don’t require action
-$status-blue:           #19b6ee !default;
-/// green, for positive status
-$status-green:          #38b44a !default;
+/// disabled state
```
Comment (anthonydillon) [inline scss/_global-settings.scss]: Can you remove this comment if there is not var for it.

File: scss/modules/_forms.scss [hunk 4]
```diff
@@ -117,7 +117,7 @@
   }
 
   form input[type='submit'] {
-    @include vf-font-size (16);
+    font-size: 16em;
```
Comment (anthonydillon) [inline scss/modules/_forms.scss]: You mean 1em

File: scss/modules/_grid.scss [hunk 5]
```diff
@@ -53,6 +53,11 @@
     font-size: 14px;
   }
 
+  .inner-wrapper {
+    margin: 0 auto;
+    max-width: 1030px;
```
Comment (anthonydillon) [inline scss/modules/_grid.scss]: If your adding this could you create a var for the max-width please.

File: scss/modules/_header.scss [hunk 6]
```diff
@@ -18,7 +18,7 @@
     z-index: 2;
     
     h2 {
-      @include vf-font-size (25);
+      font-size: 25em;
```
Comment (anthonydillon) [inline scss/modules/_header.scss]: Wow that's a big title!?!

File: scss/modules/_tables.scss [hunk 7]
```diff
@@ -6,7 +6,7 @@
     border-spacing: 0;
     overflow-x: scroll;
     margin-bottom: $gutter-width;
-    margin: 0 0 (40/$base) + em;
+    margin: 0 0 (40/16)+em;
```
Comment (anthonydillon) [inline scss/modules/_tables.scss]: Can we do the math for sass here?

File: scss/modules/_typography.scss [hunk 8]
```diff
@@ -142,7 +136,7 @@
   ul + h4,
   ol + h4,
   pre + h4 {
-    margin-top: (19.5 / $base)+em;
+    margin-top: (19.5 / 16)+em;
```
Comment (anthonydillon) [inline scss/modules/_typography.scss]: Can we do the math?

File: scss/modules/_typography.scss [hunk 9]
```diff
@@ -167,7 +161,7 @@
   }
 
   .intro {
-    @include vf-font-size (16);
+    font-size: 16em;
```
Comment (anthonydillon) [inline scss/modules/_typography.scss]: 1em probs

**Comment by anthonydillon** [comment]
Good stuff. Few comments in code.

**Comment by deadlight** [comment]
Yeah. I saw those as I was closing up. Some find+replace issues.

**Pull request merged**